app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailPresenter.java

```diff
@@ -46,12 +48,27 @@ public void attach(IEventDetailView eventDetailView, long initialEventId) {
         this.initialEventId = initialEventId;
     }
 
+    private void loadAll(boolean refresh) {
```

Comment:
>we have this complex way to load both attendees and event which will change once you complete #148 

Are we going for that structure? I thought we decided to keep present structure.